recipe/meta.yaml

```diff
@@ -19,6 +17,7 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
+    - extension-helpers
   host:
```

Comment:
Try putting extension-helpers in host

Reply:
Not totally sure that will fix anything...

Reply:
Ah yes if you have a separate build and host section I think it needs to go in host

Reply:
Thanks! I moved it to host. But if it still fails, I try sticking it in both.